console/src/main/java/org/eclipse/rdf4j/console/Util.java

```diff
@@ -118,4 +119,47 @@ public static String getPrefixedValue(Value value, Map<String,String> namespaces
 		}
 		return NTriplesUtil.toNTriplesString(value);
 	}
+	
+	/**
+	 * Format a string of values, starting new line(s) when the joined values exceed the width. 
+	 * Primarily used for displaying formatted help (e.g namespaces, config files) to the console.
+	 * 
+	 * @param width maximum column width
+	 * @param padding left padding
+	 * @param str joined string
+	 * @param separator value separator
```

Comment:
Input value separator (used to separate the str parameter) would normally be a "," or ", ", maybe I should add another parameter to specify the separator for (re-)joining the values to the formatted string

(I could pass a Set or array instead of a string, but that would require extra code in some other areas. I.e. the list of known namespaces prefixes is stored as a comma-separated value in a .properties file.)

Reply:
You’ve kind of recreated apache commons WordUtils.wrap? But with a more specific use case. Don’t know if this is particularly good or bad, but maybe a test or two would be nice as documentation.

Reply:
There is a small test in UtilTest (though there could be more tests)

I could replace the StringBuilder and the for loop by WordUtils.wrap, using "\n" + padding as the newline Str parameter.
That would also require adding commons-text as a dependency, since the wordutils.wrap in commons-lang 2 doesn't have a wrapOn parameter (e.g on ","), but that isn't a bad thing.

Reply:
Wouldn't wrap on " " (whitespace) work? I presume your list is something like ````a, b, c```` and not ````a,b,c````

Reply:
Currently it can be both, the list of templates is

`file1, file2`

and it could be even user defined templates with file names containing spaces (not a good idea IMHO, and may break further down the road, but not impossible)

And the namespace setting (which could also benefit from this formatting) uses

`prefix1 namespace1,prefix2 namepace 2...`

Reply:
Suggestion is to add this like it is now, and to replace it when upgrading to commons-lang3 (rdf4j/#1173)

Reply:
Added a deprecated annotation in order not to forget this suggestion :-)